src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.ts

```diff
@@ -58,6 +61,7 @@ export class SavedScenariosComponent implements OnInit {
   }
 
   private pollForChanges() {
+    return;
```

Comment:
This is just for testing, so my CSS inspection doesn't get overridden everytime this reloads